Full-Tortuga/decide-full-tortuga-admin
# Feature/#5 react app config

## changes
- added basic react app structure
    - router
    - api
    - types
    - utils
    - styles
    - ...

## test
- install and build the app `cd frontend && npm install && npm run build`
- check the login page for now
- check the code and implementations
